src/healthcareapis/azext_healthcareapis/generated/commands.py

```diff
@@ -29,6 +29,13 @@ def load_command_table(self, _):
         g.custom_command('delete', 'healthcareapis_service_delete', supports_no_wait=True, confirmation=True)
         g.custom_wait_command('wait', 'healthcareapis_service_show')
 
+    with self.command_group('healthcareapis acr', healthcareapis_service, client_factory=cf_service,
+                            is_experimental=True) as g:
+        g.custom_command('list', 'healthcareapis_acr_list')
+        g.custom_command('add', 'healthcareapis_acr_add', supports_no_wait=True)
+        g.custom_command('reset', 'healthcareapis_acr_reset', supports_no_wait=True)
+        g.custom_command('remove', 'healthcareapis_acr_remove', supports_no_wait=True)
+
```

Comment:
You need to add a `wait` command if you have some commands with `supports_no_wait=True`. check the doc [here](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md)